sitebot/ngBot.tcl

```diff
@@ -252,13 +252,13 @@ namespace eval ::ngBot {
 			switch -- [catch {${plugin}::init} error] {
 				-1 {
 					catch {${plugin}::deinit}
-					catch {namespace delete $plugin}
+					catch {${ns}::deinit_plugin $plugin}
 				}
 				0 { lappend loaded [namespace tail $plugin] }
 				default {
-					putlog "\[ngBot\] [namespace tail $plugin] Error :: Unable to load plugin: $error"
+					putlog "\[ngBot\] [namespace tail $plugin] Error :: Unable to load plugin: $error\n$::errorInfo"
```

Comment:
Why removing the errorInfo?

Reply:
where is  removed? i have added \n$::errorInfo for more information on error

Reply:
Ah sorry, my mixup